AllReadyApp/Web-App/AllReady/Services/TaskAttachmentService.cs

```diff
@@ -11,15 +11,15 @@
 
 namespace AllReady.Services
 {
-    public class AttachmentService : IAttachmentService
+    public class TaskAttachmentService : ITaskAttachmentService
     {
         private const string ContainerName = "attachments";
         private const int MaxAttachmentBytes = 2 * 1024 * 1024; // 2MB
         private static readonly IList<string> AllowedExtensions = new List<string> { ".png", ".jpg", ".doc", ".docx", ".xls", ".xlsx", ".pdf" };
```

Comment:
please delete MaxAttachmentBytes` and `AllowedExtensions` b/c they're not longer used.